guam/demo/DEMOGUI1.BAT

```diff
@@ -1 +1 @@
-asmlg demo\demogui1 gui test trace
+bat\asmlg guam\demo\demogui1 guam test trace
```

Comment:
If you are positioned in the z390 directory, then this is correct. Is the invocation of DEMOGUI1.BAT done then?

Just asking.
```

Reply:
I think that for all bat files we have been assuming they'd be invoked from the z390 root dir.
I have applied the same assumption to this script. Would it be more robust and flexible if use something like ~dps0~
(Sorry, I do not have exact syntax available right now. I hope you get the intention)

Reply:
The only references to DEMOGUI1 I found are in rt/RTGUAM.BAT, guam/demo/DEMOGUI1.MLC and guam/demo/DEMOGUI2.MLC. There is no invocation of rt/DEMOGUI1.BAT??? Why are you even looking at it?

As an asisde, the GUAM tests are currently not part of our regression tests. To my knowledge, few of the bat files in the rt directory and subdirectories have been looked at for the current z390.

Reply:
I think I need to run all tests, not just the ones that were included in the automated test suite. Compiling zCics has always been a major touchstone for z390. It is okay not to have these tests run every push, but I do want to be able to run these tests when needed. So yes, there is a procedure that calls it, but it is not part of z390. For now it lives outside z390 on my laptop. I have at times considered proposing a change to the build procedure to add a *ALL parameter, but have not yet done that.